components/camel-azure-eventhubs/pom.xml

```diff
@@ -121,5 +112,46 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+          <id>uber</id>
+          <build>
+            <plugins>
+              <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>5.1.1</version>
+                <executions>
+                  <execution>
+                    <id>bundle</id>
+                    <goals>
+                      <goal>bundle</goal>
+                    </goals>
+                    <configuration>
+                        <classifier>uber</classifier>
+                        <instructions>
+                            <Export-Package>
+                                org.apache.camel.component.azure.eventhubs
+                            </Export-Package>
+                            <Import-Package>
+                                kotlin*;resolution:=optional,
```

Comment:
And these, also with bnd/maven-bundle-plugin?

Reply:
Yes. Instead of using `Private-Package`, I can show how to use `Embed-Dependencies` using dependencies defined in the `pom.xml`.